.travis.yml

```diff
@@ -53,12 +53,15 @@ before_install:
   - export PR=https://api.github.com/repos/$TRAVIS_REPO_SLUG/pulls/$TRAVIS_PULL_REQUEST
   - export BRANCH=$(if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then echo $TRAVIS_BRANCH; else echo $TRAVIS_PULL_REQUEST_BRANCH; fi)
   - export COMMIT=$(if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then echo $TRAVIS_COMMIT; else echo $TRAVIS_PULL_REQUEST_SHA; fi)
-  # Installing hirak/prestissimo allows composer to parallelise downloads which speeds up the composer install.
-  - composer global require hirak/prestissimo
-  - echo "TRAVIS_BRANCH=$TRAVIS_BRANCH, PR=$PR, BRANCH=$BRANCH, COMMIT=$COMMIT"
-  - if [ "$TEST_SUITE" = "install_update" ]; then composer require goalgorilla/open_social:2.0.0 --prefer-dist; fi
+  - export TRAVIS_SLUG=$TRAVIS_REPO_SLUG
+  - export TRAVIS_PR_SLUG=$TRAVIS_PULL_REQUEST_SLUG
+
+  - echo "TRAVIS_BRANCH=$TRAVIS_BRANCH, PR=$PR, BRANCH=$BRANCH, COMMIT=$COMMIT, TRAVIS_SLUG=$TRAVIS_SLUG, TRAVIS_PR_SLUG=$TRAVIS_PR_SLUG"
+  - if [ "$TEST_SUITE" == "install_update" ]; then composer require goalgorilla/open_social:3.0.0 --prefer-dist; fi
+  - if [ "$TRAVIS_PULL_REQUEST" == "false" ] && [ "$TRAVIS_REPO_SLUG" != "goalgorilla/open_social" ]; then composer config repositories.social git https://github.com/$TRAVIS_REPO_SLUG.git; fi
   - if [ "$TRAVIS_PULL_REQUEST" != "false" ] && [ "$TRAVIS_PULL_REQUEST_SLUG" != "goalgorilla/open_social" ]; then composer config repositories.social git https://github.com/$TRAVIS_PULL_REQUEST_SLUG.git; fi
-  - if [ "$TEST_SUITE" != "install_update" ]; then composer require goalgorilla/open_social:dev-${BRANCH}#${COMMIT} --prefer-dist; fi
+  - if [ "$TEST_SUITE" != "install_update" ] && [ "$TRAVIS_REPO_SLUG" == "goalgorilla/open_social" ]; then composer require goalgorilla/open_social:dev-${BRANCH} --prefer-dist; fi
+  - if [ "$TEST_SUITE" != "install_update" ] && [ "$TRAVIS_REPO_SLUG" != "goalgorilla/open_social" ]; then composer require goalgorilla/open_social:dev-${BRANCH} --prefer-dist; fi
```

Comment:
I think you can remove this last one and the check ` && [ "$TRAVIS_REPO_SLUG" == "goalgorilla/open_social" ];` since you already change the repository in line 61 if the slug is not goalgorilla/open_social.